### PR TITLE
creating group also creates members/friends to db

### PR DESCRIPTION
**Pull request by Tome94**
made a create member function in database used that function in api create a new group

### CODE REVIEW

File: backend/src/server.ts [hunk 1]
```diff
@@ -109,14 +109,22 @@ app.post("/api/groups", async (req, res) => {
...
     user.id,
     subscription.name,
     subscription.cost,
     new Date(),
     subscription.image
   );
-
+  for (const memberData of friends) {
+    const createdFriend = await createMember(
+
```
Comment (Winxton) [inline backend/src/server.ts]: just missing the group ID to attach it to the group

**Review by Winxton — COMMENTED** [review]
LGTM after changes!

File: backend/src/server.ts (hunk 1 above)
Comment (Tome94) [reply backend/src/server.ts]: not sure if this is the way to do it.

Comment (Winxton) [reply backend/src/server.ts]: Looks like it'll work! Did you test it?

Comment (Tome94) [reply backend/src/server.ts]: When I run the backend I encounter a network error some kind of authentication error from supabase this was before I did any changes. Check discord.

**Pull request merged**